pkg/types/conversion/installconfig.go

```diff
@@ -176,6 +176,18 @@ func convertOpenStack(config *types.InstallConfig) error {
 		return err
 	}
 
+	// machinesSubnet has been deprecated in favor of ControlPlanePort
+	controlPlanePort := config.Platform.OpenStack.ControlPlanePort
+	deprecatedMachinesSubnet := config.Platform.OpenStack.DeprecatedMachinesSubnet
+	if deprecatedMachinesSubnet != "" && controlPlanePort == nil {
+		fixedIPs := make([]openstack.FixedIP, 1)
+		fixedIPs[0].Subnet.ID = deprecatedMachinesSubnet
+		config.Platform.OpenStack.ControlPlanePort = &openstack.PortTarget{FixedIPs: fixedIPs}
```

Comment:
Should this emit some warning?

Reply:
I was following the same pattern done for upconvertVIP, but I can add a warning if you think it's worth. Let me know what you think

Reply:
Fair enough, it can be a separate PR if this is a pattern already.